lib/standalone.cjs

```diff
@@ -126,6 +126,8 @@ async function standalone({
 				? path.join(cwd, codeFilename)
 				: codeFilename;
 
+		formatterFunction = await getFormatter(stylelint, { filePath: absoluteCodeFilename });
```

Comment:
What happens if `absoluteCodeFilename` is falsy?
e.g. if `codeFilename` is `undefined`